docs/current/sailbot_workspace/setup.md

```diff
@@ -171,31 +196,33 @@ git clone https://github.com/UBCSailbot/sailbot_workspace.git
 ## 5. Open Sailbot Workspace in a Dev Container
 
 1. Ensure that Docker is running
-2. Run the `Dev Containers: Reopen in Container` command in the
-   [VS Code command palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette){target=_blank}
+2. Run the `Dev Containers: Reopen in Container` command
 
-## 6. Open the VS Code workspace file
+## 6. Open the workspace file
 
 1. Open the file `.devcontainer/config/sailbot_workspace.code-workspace` in VS Code
 2. Click `Open Workspace`
 
-## 7. Run the `setup` VS Code task
+## 7. Run the `setup` task
 
 !!! info ""
 
     Moved from Run to Setup page in [:octicons-tag-24: v1.1.0](https://github.com/UBCSailbot/sailbot_workspace/releases/tag/v1.1.0){target=_blank}
 
-The `setup` task clones the repositories defined in [`src/new_project.repos`](https://github.com/UBCSailbot/sailbot_workspace/blob/main/src/new_project.repos){target=_blank}
-and updates dependencies of the ROS packages.
+The `setup` task clones the repositories defined in `src/new_project.repos` and updates dependencies of the ROS packages.
+
+??? bug "Can't see the `setup` task"
+
+    If you can't see the `setup` task, run the `Developer: Reload Window` command.
+    This may occur when the workspace file is opened for the first time.
 
-1. Run the `Tasks: Run Task` command in the
-   [VS Code command palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette){target=_blank}
-2. Select `setup` from the dropdown menu
+## 8. Run the `Build All` task
 
-    ??? bug "Can't see the `setup` task"
+The `Build All` task builds all the ROS packages.
```

Comment:
Should we add a comment when the `Build All` package does not build some of the packages such as the network systems and the custom interfaces repositories?

Reply:
This should not happen, if it does it is a bug. Has this happened for you recently?

Reply:
As per my previous suggestion, link to the section describing how to run VS code commands (if you go with that suggestion).